lighthouse-core/config/config.js

```diff
@@ -437,9 +437,15 @@ class Config {
       assertValidPluginName(configJSON, pluginName);
 
       const pluginPath = resolveModule(pluginName, configDir, 'plugin');
+
       const rawPluginJson = require(pluginPath);
       const pluginJson = ConfigPlugin.parsePlugin(rawPluginJson, pluginName);
 
+      if (rawPluginJson['localePath']) {
```

Comment:
naming bikeshed time?!!?!

first inclination is to call it `localesPath` (since theres a lot of files in there)

but i wonder if there's a more industry standard term for a directory full of i18n strings.  @exterkamp ?

Reply:
switched to `localesPath` for now

Reply:
Now `localeStrings` due to updated config